EKF/estimator_base.h

```diff
@@ -320,5 +341,25 @@ class EstimatorBase
 
 	uint64_t _last_gps_origin_time_us = 0;
 	struct map_projection_reference_s _posRef = {};
-	float _gps_alt_ref = 0.0f;
+    float _gps_alt_ref = 0.0f;
```

Comment:
@priseborough Do those variables need to be public. Otherwise I would move them up. They are a bit lonely down there.